doc/explanation/scopes.rst

```diff
@@ -0,0 +1,36 @@
+Dune projects and workspaces
```

Comment:
```suggestion
Dune Projects and Workspaces
```

Reply:
We use title case everywhere expect with headers that are about dune files like `dune` and `dune-project`.